.travis.yml

```diff
@@ -51,7 +51,7 @@ install:
   # Ensure pip and nose will install within the conda env
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip nose numpy matplotlib pandas
   - source activate test-environment
-  - pip install coverage codecov                           # code coverage; see other projects on how to install w/Miniconda
```

Comment:
Note: it is important here to install `coverage` before `codecov` for python v3.5.

Just like you have it :)